### DISCUSSION TITLE
Authorization

### DISCUSSION THREAD

**Discussion by jmargutt** [rodekruis/ADA-API#16] [General]
@gulfaraz maybe this is trivial, but I understand that for authorization you just need headers = {'Authorization': 'Bearer 0Ar0EjumFD'} in the request, however I get a 403 forbidden resource, is there something that I'm missing?

**Comment by gulfaraz** [comment]
@jmargutt Which end-point responds with 403? Are you getting this on swagger or python script?

**Reply by jmargutt** [reply]
@gulfaraz both on swagger and python script, tried both POST events and PATCH events/{id}

**Reply by gulfaraz** [reply]
I get a 201 response for the POST request and a 200 response for the PATCH request. Can you please double-check the Bearer token? It should be the ADMIN_CODE env variable.

**Reply by jmargutt** [reply]
It's not clear to me how to use this, I think it's faster if you show me at the standup

**Reply by gulfaraz** [reply]
I've sent the code via WhatsApp

**Reply by jmargutt** [reply]
OK this works thanks
